apluslms_roman/cli.py

```diff
@@ -346,9 +383,10 @@ def build_action(context):
         steps = builder.get_steps()
         num_len = max(2, len(str(len(steps)-1)))
         name_len = max(4, len(max(step.name or "" for step in steps)))
+        name_len = max(4, max(len(s.name or "") for s in steps))
```

Comment:
Replaces above line, but that old line is not removed.

Reply:
....I fixed that at some point